### ISSUE TITLE
[BUG] @W-9296240@ Commands result in error "Unable to determine a suitable edition, even after broadening."

### ISSUE THREAD

**Issue by jbartolotta-sfdc** [forcedotcom/sfdx-scanner#412]
**Behavior**
Invoking the Salesforce CLI `sfdx-scanner` command may result in the error `Unable to determine a suitable edition, even after broadening.`

**Cause**
Versions `2.6.0-2.8.0` of the `sfdx-scanner` plugin have a dependency on a library that became inoperable with the latest version of the Salesforce CLI.

**Workaround**
We are in the process of fixing the issue. In the meantime, you can workaround this issue by installing version `2.5.1` of the `sfdx-scanner` plugin.

```bash
# The following commands will uninstall the current version and install version 2.5.1
> sfdx plugins:uninstall @salesforce/sfdx-scanner
> sfdx plugins:install @salesforce/sfdx-scanner@2.5.1
```

Alternatively you can use sfdx versions `2.6.0-2.8.0` with any Node.js version other than 16 by following the instructions found [here](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_npm).

**How will this be avoided in the future?**
We are working to improve our processes to detect such incompatibilities earlier to avoid such issues in the future. Sorry for any inconvenience.

**Comment by ranekere-sfdc** [comment]
Bug is fixed in 2.9.1. Please upgrade the plugin.

**Comment by jbartolotta-sfdc** [comment]
This issue has been fixed in version `2.9.1`. Users experiencing this issue will need to uninstall the existing plugin and reinstall the latest version with the following commands.

```bash
> sfdx plugins:uninstall @salesforce/sfdx-scanner
> sfdx plugins:install @salesforce/sfdx-scanner
```
